canonicalwebteam/blog/django/views.py

```diff
@@ -55,6 +76,21 @@ def group(request, slug, template_path):
     return render(request, template_path, context)
 
 
+def group_feed(request, slug, title=blog_title, subtitle=""):
+    try:
+        feed = blog_views.get_group_feed(
+            group_slug=slug,
+            uri=request.build_absolute_uri(),
+            title=title,
+            subtitle=subtitle,
+            article_url_generator=article_url_generator,
+        )
+    except Exception as e:
+        return HttpResponse("Error: " + str(e), status=502)
```

Comment:
I know you're just following precedent here, but I really dislike this pattern. I assume this is just so that we pass a `502` rather than a `500` status? But I really don't think we necessarily know if that's accurate. The error could be coming from anywhere, in our code, or whatever. I think we should only return e.g. `502` in really specific circumstances where we definitely know that it's wordpress's fault.

Would you mind not adding more of these `try..except` blocks in new functions? Just let errors bubble up to be handled by the application as normal please.

Reply:
I agree and would like to remove it in all views, since this is actually masking possible errors(we don't get them in sentry). I can do this now for this views, or do them all in a separate PR?

Reply:
Or do them all in this PR? I don't mind, up to you, but please just don't add more of them =)